code/modules/mob/living/carbon/human/species.dm

```diff
@@ -462,6 +462,29 @@
 	H.remove_overlay(MOTH_WINGS_LAYER)
 	H.remove_underlay(MOTH_WINGS_BEHIND_LAYER)
 
+/datum/species/sectoid
+	name = "Sectoid"
+	name_plural = "Sectoids"
+	icobase = 'icons/mob/human_races/r_sectoid.dmi'
+	deform = 'icons/mob/human_races/r_sectoid.dmi'
+	default_language_holder = /datum/language_holder/moth
```

Comment:
```suggestion
	default_language_holder = /datum/language_holder/sectoid
```

Reply:
Moth speak and common are included in the moff holder, just make a 

/datum/language_holder/sectoid
	languages = list(/datum/language/sectoid)

in language_holder.dm